client/my-sites/plans/ecommerce-trial/controller.jsx

```diff
@@ -10,6 +11,14 @@ export function trialExpired( context, next ) {
 	const state = context.store.getState();
 	const selectedSite = getSelectedSite( state );
 
+	context.store.dispatch(
+		recordTracksEvent( 'calypso_plan_trial_expired_page', {
+			site_id: selectedSite.ID,
```

Comment:
There's already a `blog_id` event prop recorded, we probably don't need to add another here. Same for other two events.